sympy/polys/orthopolys.py

```diff
@@ -144,6 +155,22 @@ def hermite_poly(n, x=None, polys=False):
     """
     return named_poly(n, dup_hermite, ZZ, "Hermite polynomial", (x,), polys)
 
+@public
+def hermite_prob_poly(n, x=None, polys=False):
```

Comment:
I guess if this is public it should be expose at top level import just like `hermite_poly`.